plugins/flashrom/fu-plugin-flashrom.c

```diff
@@ -39,7 +40,8 @@ fu_plugin_init (FuPlugin *plugin)
 	fu_plugin_set_build_hash (plugin, FU_BUILD_HASH);
 	fu_plugin_add_rule (plugin, FU_PLUGIN_RULE_METADATA_SOURCE, "linux_lockdown");
 	fu_plugin_add_rule (plugin, FU_PLUGIN_RULE_CONFLICTS, "coreboot"); /* obsoleted */
-	fu_plugin_add_flag (plugin, FWUPD_PLUGIN_FLAG_REQUIRE_HWID);
+	fu_plugin_set_device_gtype (plugin, FU_TYPE_FLASHROM_LSPCON_I2C_SPI_DEVICE);
```

Comment:
I think if you're adding this code, don't we also need one for internal?

Reply:
Internal is called for coldplug method on the plugin

Reply:
It's a bit more tricky than that, it needs to be g_type_ensure if it's being created from a quirk. I think fu_plugin_**set**_device_gtype means we can't create internal ones anymore too.

Reply:
In my testing both device types coexist...